lib/polymer/iron.dart

```diff
@@ -249,18 +187,36 @@ class CoreElement extends WebElement {
 
     return _eventStreams[eventName];
   }
+
+  void add(dynamic child) {
+    if (child is WebElement) {
+      child = child.element;
+    } else if (child is Element) {
+      child = child;
+    } else {
+      throw new ArgumentError('child must be a WebElement or an Element');
+    }
+    //Polymer.dom(this).appendChild(child);
+    context["Polymer"].callMethod("dom", [new JsObject.fromBrowserObject(element)])
+    .callMethod("appendChild", [new JsObject.fromBrowserObject(child)]);
+  }
+
+  dynamic selectorAll(String selector) {
+    //Polymer.dom(this).childNodes;
+    return context["Polymer"].callMethod("dom", [new JsObject.fromBrowserObject(element)]).callMethod("querySelectorAll", [selector]);
+  }
 }
 
 class Transition {
-  static void coreTransitionCenter(CoreElement element) =>
+  static void coreTransitionCenter(PolymerElement element) =>
       element.setAttribute('transition', 'core-transition-center');
 }
 
 class Transitions {
-  static void slideFromRight(CoreElement element) =>
+  static void slideFromRight(PolymerElement element) =>
```

Comment:
Yeah, I still need to update those transitions. I have looked at those neon animation. I use it for the dropdown animation (in the right top). Will try to get the slidefromRight transition working. I also need to get the riddles in the left menu back.